submissions/krylenger/dom-api-task/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Motorcycles</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Abel&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <input id='nav__toggle' type="checkbox" class="nav__toggle">
+    <label for="nav__toggle" class="nav__toggle-label"></label>
+    <nav class="nav">
+        <ul class="nav__ul">
+            <li class="nav__li"><a class="nav__link">ADVENTURE</a></li>
+            <li class="nav__li"><a class="nav__link">NAKED</a></li>
+            <li class="nav__li"><a class="nav__link">CRUISER</a></li>
+            <li class="nav__li"><a class="nav__link">TOURING</a></li>
+            <li class="nav__li"><a class="nav__link">SPORT</a></li>
+            <li class="nav__li"><a class="nav__link">SUPERMOTO</a></li>
+            <li class="nav__li"><a class="nav__link">OFFROAD</a></li>
+            <li class="nav__li"><a class="nav__link">RETRO</a></li>
+        </ul>
+    </nav>
+    <main class="main">
+        <h1 class="main__header"> Choose your bike</h1>
+        <div class="main__img-container">
+            <img src="./images/logo.png" alt="img" class="main__img">
+        </div>
+        <div class="main__description">
+
+        </div>
+        <ul class="main__ul-bikes">
+
+        </ul>
+        <p class="main__avg-price"></p>
+    </main>
+</body>
+<script src="./script.js"></script>
```

Comment:
It's better to place this `script` before the `body` tag closing

Reply:
✅ Done